### DISCUSSION TITLE
Using data channel on videojs-webrtc-plugin

### DISCUSSION THREAD

**Discussion by massimilianoboggio** [ant-media/Ant-Media-Server#4981] [Q&A]
Hi everyone,
I am happy using the videojs plugin and would like to send and receive messages via the WebRTC Data Channel in my stream. Is it possible to do this with the videojs-webrtc-plugin?
thk so much

**Comment by USAMAWIZARD** [comment]
Hi 
we do not support data channel with  videojs-webrtc-plugin as of now , but it will be a nice feature to have I have created a github issue regarding that. https://github.com/ant-media/Ant-Media-Server/issues/4990
